lib/dev/commands/cd.rb

```diff
@@ -6,7 +6,7 @@ module Commands
       basename = RUBY_PLATFORM =~ /darwin/ ? 'fzy_darwin' : 'fzy_linux'
       File.expand_path("vendor/#{basename}", ROOT)
     end
-    GITHUB_ROOT = '~/src/github.com'
+    GITHUB_ROOT = ENV.fetch('DEV_GITHUB_ROOT', '~/src/github.com')
```

Comment:
There's another reference to `~/src/github.com` in `clone.rb` that you probably should handle as well.

Reply:
🤔 This just gets more complicated. If I want to DRY the code, there is a philosophical debate between using env or some kinds configuration file (eg/ yaml).

What are your thoughts on creating a new module for configuration and parse the github_root from there. So something like

```ruby
# in lib/dev/config.rb

module Dev
  module Config
     def self.github_root
        # read from env example:
           ENV.fetch('DEV_GITHUB_ROOT', '~/src/github.com')

        # or parse some kind of yaml file and load the key here
     end
  end
end
```

I am personally OK with using environment variable to keep it simple. However, we could have something like `~/.dev/config.yaml`.

Reply:
That's a good point about dev config set.

I have updated the code, not sure if @burke is OK with the direction.